djangocms_fil_bootstrap/__init__.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+__version__ = "0.0.0"
+
+
+def bootstrap():
+    from .bootstrap import Bootstrap
+    from .components import Collections, Groups, Pages, Permissions, Users, Workflows
+
+    bootstrap = Bootstrap(
+        os.path.join(os.path.dirname(os.path.abspath(__file__)), "data.json")
+    )
+    bootstrap(Users, Groups, Permissions, Pages, Workflows, Collections)
```

Comment:
In my opinion this should be __init__ of a class and line 11 should be __call__.